pepper-apis/dss-core/pom.xml

```diff
@@ -394,15 +394,12 @@
         <dependency>
             <groupId>redis.clients</groupId>
             <artifactId>jedis</artifactId>
+            <version>${jedis.version}</version>
```

Comment:
No change to version, but put it in a central place for better future management.